src/run.ts

```diff
@@ -5,82 +5,135 @@ import { TERMINAL_NAME, PYSHINY_EXEC_CMD } from "./extension";
 import { AddressInfo } from "net";
 import { getRemoteSafeUrl } from "./extension-api-utils/getRemoteSafeUrl";
 
+const DEBUG_NAME = "Debug Shiny app";
```

Comment:
Is this shown as a context somewhere on the VSCode IDE?

Should we rename to `Debugging Shiny app`?

Reply:
It's not currently, but if it were, it's supposed to be a verb as it would be presented as an option for the user to select when launching a debugging job.